testing/src/main/java/org/oppia/android/testing/network/BUILD.bazel

```diff
@@ -41,4 +41,19 @@ kt_android_library(
     ],
 )
 
+kt_android_library(
+    name = "retrofit_module",
+    testonly = True,
```

Comment:
Should I also change the name of the library to `retrofit_test_module`?

Reply:
'test_module' is sufficient, I think (that's what we conventionally do elsewhere).

Reply:
Ack.